ios/FluentUI.Demo/FluentUI.Demo/TableViewSampleData.swift

```diff
@@ -25,9 +26,25 @@ class TableViewSampleData {
         let hasCustomLeadingView: Bool
         let hasHandler: Bool
 
-        init(title: String, items: [Item] = [], numberOfLines: Int = 1, hasFullLengthLabelAccessoryView: Bool = false, hasAccessory: Bool = false, accessoryButtonStyle: TableViewHeaderFooterView.AccessoryButtonStyle = .regular, allowsMultipleSelection: Bool = true, headerStyle: TableViewHeaderFooterView.Style = .header, hasFooter: Bool = false, footerText: String = "", footerLinkText: String = "", hasCustomLinkHandler: Bool = false, hasCustomAccessoryView: Bool = false, hasCustomLeadingView: Bool = false, hasHandler: Bool = false) {
+        init(title: String,
+             items: [Item] = [],
+             isUnreadDotVisible: Bool = false,
+             numberOfLines: Int = 1,
+             hasFullLengthLabelAccessoryView: Bool = false,
+             hasAccessory: Bool = false,
+             accessoryButtonStyle: TableViewHeaderFooterView.AccessoryButtonStyle = .regular,
+             allowsMultipleSelection: Bool = true,
+             headerStyle: TableViewHeaderFooterView.Style = .header,
+             hasFooter: Bool = false,
+             footerText: String = "",
+             footerLinkText: String = "",
+             hasCustomLinkHandler: Bool = false,
+             hasCustomAccessoryView: Bool = false,
+             hasCustomLeadingView: Bool = false,
+             hasHandler: Bool = false) {
             self.title = title
             self.items = items
+            self.isUnreadDotVisible = isUnreadDotVisible
```

Comment:
Nit: you call it the leadingDot elsewhere, should we be consistent with naming?

Reply:
Yes I totally thought I had replaced everything but clearly not! Changed it!